app/services/MeansOfTransportIdentificationTypesService.scala

```diff
@@ -65,6 +65,17 @@ class MeansOfTransportIdentificationTypesService @Inject() (
           )
     }
 
+  def getMeansOfTransportIdentificationType(userAnswersCode: Option[String])(implicit hc: HeaderCarrier): Future[Option[TransportMeansIdentification]] =
```

Comment:
Does this need to be an option? Looks like it's only used in the transformer where you're wrapping the value in a Some